assembly/test/channels.ts

```diff
@@ -1,6 +1,5 @@
-import { Console } from "as-wasi";
-import { Channel } from "lunatic";
-import { getIP, tryParseIPV4 } from "../assembly/lunatic-util/string";
+import { Channel } from "..";
+import { getIP, tryParseIPV4 } from "..";
```

Comment:
Ah. Moving the tests into the `assembly` folder is probably a good idea. Thanks for the suggestion.

Reply:
Yeah though asbuild allows for workspaces so it might not be a bad idea to use that, then `asb` would build all of them.